starboard/android/apk/app/src/main/java/dev/cobalt/coat/MediaPlaybackService.java

```diff
@@ -82,7 +83,12 @@ public void onDestroy() {
   public void startService() {
     if (this.channelCreated) {
       try {
-        startForeground(NOTIFICATION_ID, buildNotification());
+        if (VERSION.SDK_INT >= 29) {
+          startForeground(
+              NOTIFICATION_ID, buildNotification(), ServiceInfo.FOREGROUND_SERVICE_TYPE_MANIFEST);
```

Comment:
would we use FOREGROUND_SERVICE_TYPE_MEDIA_PLAYBACK here to avoid changing the AndroidManefest.xml for both CoAT and Kimono?